.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Rust
```

Comment:
Renamed to `main`, it's the name used for the badge by default (I didn't find how to change it... and the change seems reasonable)